common/network-shuffle/src/test/java/org/apache/spark/network/shuffle/RemoteBlockPushResolverSuite.java

```diff
@@ -1612,19 +1612,8 @@ private void verifyMetrics(
     assertEquals(expectedIgnoredBlocksBytes, ignoredBlockBytes.getCount(), "ignored block bytes");
   }
 
-  private static class PushBlock {
-    private final int shuffleId;
-    private final int shuffleMergeId;
-    private final int mapIndex;
-    private final int reduceId;
-    private final ByteBuffer buffer;
-    PushBlock(int shuffleId, int shuffleMergeId, int mapIndex, int reduceId, ByteBuffer buffer) {
-      this.shuffleId = shuffleId;
-      this.shuffleMergeId = shuffleMergeId;
-      this.mapIndex = mapIndex;
-      this.reduceId = reduceId;
-      this.buffer = buffer;
-    }
+  private record PushBlock(
+      int shuffleId, int shuffleMergeId, int mapIndex, int reduceId, ByteBuffer buffer) {
```

Comment:
These fields should be `private`, shouldn't these?

Reply:
Ok, let me revert this one. The previous commit did not revert it due to this in test case.

Reply:
done